trigger/test-timer/README.md

```diff
@@ -0,0 +1,103 @@
+<!--
+title: CSV Timer
+
+-->
+# Timer Trigger
+This trigger provides your flogo application the ability to get data from a CSV file after a particular interval for an action 
+
+## Installation
+
+```bash
+flogo install github.com/skothari-tibco/csvtrigger
+```
+
+## Metadata
+```json
+{
+  "handler": {
+    "settings": [
+      {
+        "name": "filePath",
+        "type": "string"
+      },
+      {
+        "name": "repeatInterval",
+        "type": "string"
+      }
+    ]
+  }
+}
+```
+### Details
+
+#### Settings:
+| Setting  | Required | Description |
+|:---------|:---------|:------------|
+| control  | false    | Enable control of trigger
+| port     | false    | Port number on which control api is called (/control/resume, /control/pause, /control/restart)
```

Comment:
We can add an autostart flag or something.  But definitely start/stop.  The pause/resume only make sense if you are parsing the csv file and emmitting a value every (1ms, 10ms).  So I think we should have a mode where you are parsing the csv file and emitting rows.
So you can have an emit data time interval, (1ms.. 10ms.. etc.) and a flag or enum to indicate whether to emit rows or the entire file.    

In this case of rows, pause/resume would make more sense, because otherwise they are essentially just start and stop.

Reply:
I would remove the timer stuff, this is essentially a loop for each handler with a sleep in it.  Also I would add the option of specifying a handler id when invoking the control rest commands.  That way if someone wants to pause a particular handler they can.

Reply:
If we remove the timer stuff, how will we decide, the interval after which the data should be given as output?

Reply:
simple loop with a sleep would probably suffice, you just sleep for that interval